devicetypes/smartthings/aeon-led-bulb.src/aeon-led-bulb.groovy

```diff
@@ -203,6 +203,11 @@ def setColorTemperature(temp) {
 	commands(cmds)
 }
 
+private queryAllColors() {
```

Comment:
This doesn't seem related to the PR title. Thoughts on making this a separate ticket/PR?

Reply:
Sure. Just noticed it while I was doing color dth work.

Reply:
Good catch - `setColorTemperature` must be completely broken right now so at a minimum we should get this fix out in next week's release